iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/procedure/impl/region/RemoveRegionPeerProcedure.java

```diff
@@ -100,6 +102,7 @@ protected Flow executeFromState(ConfigNodeProcedureEnv env, RemoveRegionPeerStat
           setNextState(DELETE_OLD_REGION_PEER);
           break;
         case DELETE_OLD_REGION_PEER:
+          handler.updateRegionCache(consensusGroupId, targetDataNode, RegionStatus.Removing);
```

Comment:
Why should update RegionStatus to Removing twice?

Reply:
same as above